GibbsSampling.Library/CompositeVector/CompositeVectorTypes.fs

```diff
@@ -0,0 +1,73 @@
+﻿namespace CompositeVector
+
+open System
+open BioFSharp
+
+module Types =
+    
+    /// Matrix with fixed positions for nucleotides and amino acids.
+    type BaseMatrix<'a, 'value when 'a :> IBioItem>internal(matrix:'value [,]) =
```

Comment:
This type syntax `type BaseMatrix<'a, 'value when 'a :> IBioItem>internal(matrix:'value [,])` looks extremely complex to me. If possible dump it down to increase readability.

Reply:
Created new issue:
https://github.com/Etschbeijer/GibbsSampling2.0/issues/11